### ISSUE TITLE
documenting the rmProg family

### ISSUE THREAD

**Issue by JacquesCarette** [hakaru-dev/hakaru#86]
I've been scouring the ppaml emails that I kept (not all) to find what
each rmProgi (for i = 1..4) were.  rmProg1 is 'easier roadmap', rmProg2
is obtained by disintegration (and simplification?), etc.  Seems there
was a step in there involving mh.

Now that things work a lot better, it would be nice to have this as a
'system test' to ensure that our pipeline works. I guess this is an
example in the most recent paper that Rob talked about? Could we get
enough information to turn it (and the other examples) into actual tests?

**Comment by ccshan** [comment]
Yeah, Section 3 of ppaml/writing/multi.tex should document this pipeline.  It welcomes your feedback.
